three.c

```diff
@@ -0,0 +1,67 @@
+//Matthew Kellerman -sees if user input are numbers, if they form a triangle, and what triangle do they form
```

Comment:
Nice job! 👍 the only thing I would have handled differently is float input - it truncated the values so 4.4, 4.3, 4.2 was seen as equilateral.